pyaccel/new_lifetime.py

```diff
@@ -1,7 +1,9 @@
 import pyaccel
 import pymodels
+import mathphys
 
 import numpy as np
+from scipy import integrate
 import math
 
 #Life time calculations
```

Comment:
Following the line of the last comment, all these variables should be passes as input to the function which calculates the lifetime. Maybe you could organize them in a dictionary which would be defined in a driving script, like this:
```
params = dict(
    model = pymodels.si.create_accelerator(),
    abertura_x = 12e-3,  # m
    abertura_y = 2e-3,  # m
    energy_spread = 8.7e-4,
    vac = 1e-9,  # mTorr
    I = 100e-3,  # A
    n_bunches = 864,
    natural_emit = 0.25e-9,  # m rad
    bunch_lengh = 2.3e-3,  # m
    coupling = 0.01,
    energy_accep = 0.03,
    ring_loc = 0)
```